sky/execution.py

```diff
@@ -378,55 +379,120 @@ def spot_launch(
               f'{colorama.Style.RESET_ALL}')
         return
 
+    # ================================================================
     # Translate the workdir and local file mounts to cloud file mounts.
-    new_storage_mounts = {}
+    # ================================================================
```

Comment:
UX nits: I feel L366 and L369-370 can be omitted. The former is strongly implied. The latter currently has 1 strategy only. Wdyt?